stac_fastapi/elasticsearch/stac_fastapi/elasticsearch/serializers.py

```diff
@@ -22,6 +24,30 @@ def db_to_stac(cls, item: dict, base_url: str) -> TypedDict:
 class ItemSerializer(Serializer):
     """Serialization methods for STAC items."""
 
+    @classmethod
+    def stac_to_db(cls, stac_data: TypedDict, base_url: str) -> stac_types.Item:
+        """Transform stac item to database ready stac item."""
+        item_links = ItemLinks(
+            collection_id=stac_data["collection"],
+            item_id=stac_data["id"],
+            base_url=base_url,
+        ).create_links()
+        stac_data["links"] = item_links
+
+        # elasticsearch doesn't like the fact that some values are float and some were int
```

Comment:
I'm going to fix this in #31 -- I'm about to merge the stac-server issue about the same thing, so it should be straightforward to port it.